### PR TITLE
feat: better blueprint plugin

### PR DESCRIPTION
**Pull request by pbullhove**
## What does this pull request change?

Modifies the old blueprint plugin. 

1. Better UI
2. Fixes bug of restoring that did not restore everything

image

### CODE REVIEW

File: packages/dm-core-plugins/src/blueprint/BlueprintPlugin.tsx [hunk 1]
```diff
@@ -1,231 +1,28 @@
...
-    </div>
-  )
-}
-
 export const BlueprintPlugin = (props: IUIPlugin) => {
   const { idReference } = props
   const { document, isLoading, updateDocument, error } =
     useDocument<TBlueprint>(idReference)
   const [formData, setFormData] = useState<any>({ ...document }) //TODO remove any type (requires TBlueprint to be updated)
+  const dirtyState = !isEqual(document, formData)
```
Comment (soofstad) [inline packages/dm-core-plugins/src/blueprint/BlueprintPlugin.tsx:25]: Would a "useMemo" help on performance here?

**Review by soofstad — APPROVED** [review]
Looks good! :100:

File: packages/dm-core/src/components/Pickers/BlueprintPicker.tsx [hunk 2]
```diff
@@ -33,15 +35,14 @@ export type TBlueprintPickerProps = {
   /** A title for the picker */
   label?: string
   /** Render a input element or just a button */
-  type?: 'button' | 'input'
+  type?: 'button'
```
[the remark this answers is not in the document]
Comment (pbullhove) [reply packages/dm-core/src/components/Pickers/BlueprintPicker.tsx:38]: It was not used. 

Also, it behaved really weirdly. It was an input-box that when you clicked it behave just like the button, opening the tree-view. 
So then we can just use the button.

File: packages/dm-core-plugins/src/blueprint/BlueprintPlugin.tsx (hunk 1 above)
Comment (pbullhove) [reply packages/dm-core-plugins/src/blueprint/BlueprintPlugin.tsx:25]: yes, very much! Thank you for the pointer.

**Comment by henrikbossart** [comment]
I think there's a merge that's gone wrong here :/

**Pull request merged**